force-app/main/default/classes/AccountHelper.cls

```diff
@@ -28,19 +28,70 @@ Always ensure that your helper methods are bulk-safe to handle multiple records
 */
 public with sharing class AccountHelper {
     
-    public void setTypeProspect(SObject objects){
-
+    public static void setTypeProspectAfterInsert(List<Account> newAccounts) {
+        List<Account> accountsToUpdate = new List<Account>();
+            for (Account acc : newAccounts) {
+                if (acc.Type == null) {
+                    Account accToUpdateAfterInsert = new Account(Id = acc.Id , Type = 'Prospect');
+                    accountsToUpdate.add(accToUpdateAfterInsert);
+                }
+            }
+    
+            if (!accountsToUpdate.isEmpty()) {
+                update as system accountsToUpdate;
+            }
+        }
+    
+    public static void addressCopyAfterInsert(List<Account> newAccounts) {
+        List<Account> accountsToUpdate = new List<Account>();
+        for (Account acc : newAccounts) {
+            if (acc.ShippingStreet != null ||
+                acc.ShippingCity != null ||
+                acc.ShippingState != null ||
+                acc.ShippingPostalCode != null ||
+                acc.ShippingCountry != null) {
```

Comment:
Try using the shippingaddress compound field.

Reply:
I couldn't quite figure this one out.  I tried to use acc.BillingAddress = acc.ShippingAddress (field not writeable), then adjusted to where I just used acc.ShippingAddress for the null check, but when I looped through newAccounts to update the values, they didn't copy over correctly per test.